Normalization/Application/Commands/PeriodDateFixerCommand.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Hl7.Fhir.Model;
+using LantanaGroup.Link.Normalization.Application.Models;
+using LantanaGroup.Link.Normalization.Domain.Entities;
+using LantanaGroup.Link.Shared.Application.Models.Kafka;
+using MediatR;
+
+namespace LantanaGroup.Link.Normalization.Application.Commands;
+
+public class PeriodDateFixerCommand : IRequest<OperationCommandResult>
+{
+    public Bundle Bundle { get; set; }
+    public List<PropertyChangeModel> PropertyChanges { get; set; }
+}
+
+public class PeriodDateFixerCommandHandler : IRequestHandler<PeriodDateFixerCommand, OperationCommandResult>
+{
+    public async Task<OperationCommandResult> Handle(PeriodDateFixerCommand request, CancellationToken cancellationToken)
+    {
+        var bundle = request.Bundle;
+        var propertyChanges = request.PropertyChanges;
```

Comment:
Doesn't seem like this is needed or maybe it will get some use later?

Reply:
fixed.